lib/mixins/dropdown.js

```diff
@@ -81,6 +80,9 @@ export default {
                         el.addEventListener('mouseover', this.noop);
                     });
                 }
+
+                document.addEventListener('keydown', this.keydownListener);
```

Comment:
But this is a global eventListener, so we don't need to focus on first element on each show